pkg/kubelet/util/pod_startup_latency_tracker.go

```diff
@@ -112,6 +114,13 @@ func (p *basicPodStartupLatencyTracker) ObservedPodOnWatch(pod *v1.Pod, when tim
 		metrics.PodStartSLIDuration.WithLabelValues().Observe(podStartSLOduration)
 		metrics.PodStartTotalDuration.WithLabelValues().Observe(podStartingDuration.Seconds())
 		state.metricRecorded = true
+		// if is the first Pod with network track the start values
+		// these metrics will help to identify problems with the CNI plugin
+		if !pod.Spec.HostNetwork && !p.firstNetworkPodSeen {
+			metrics.FirstNetworkPodStartSLIDuration.Set(podStartSLOduration)
+			metrics.FirstNetworkPodStartTotalDuration.Set(podStartingDuration.Seconds())
+			p.firstNetworkPodSeen = true
```

Comment:
Does this need to also account for CNI config modifications and CNI plugin changes?

Reply:
Discussed offline, the CNI is immutable, so we don't have to account for that.